internal/acceptance/cluster_test.go

```diff
@@ -59,6 +59,24 @@ func TestAccClusterResource_CreateSingleNodeCluster(t *testing.T) {
 			instance_pool_id = "{env.TEST_INSTANCE_POOL_ID}"
 			num_workers = 0
 			autotermination_minutes = 10
+			spark_conf = {
+				"spark.master" = "local[*]"
+			}
+		}`,
+	})
+}
+
+func TestAccClusterResource_CreateSingleNodeCluster(t *testing.T) {
+	workspaceLevel(t, step{
+		Template: `
+		data "databricks_spark_version" "latest" {
+		}
+		resource "databricks_cluster" "this" {
+			cluster_name = "singlenode-{var.RANDOM}"
+			spark_version = data.databricks_spark_version.latest.id
+			node_type_id = "{env.TEST_INSTANCE_POOL_ID}"
```

Comment:
you need to look this up from a data source, not using the instance pool id
integration tests failed with `Node type *** is not supported`

Reply:
Can you clarify what data source you are talking about? Is it a variable or something?

Reply:
@nkvuong ping?